### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by agusaldasoro**
# Summary
* Provide the Bootstrap stylesheets.

# Test Plan

# Trello Card
https://trello.com/c/Zkb7505U

# Screenshots

### CODE REVIEW

File: blog/app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: [:show, :edit, :update, :destroy]
+
+  # GET /posts
```
Comment (JesusHerrera) [inline blog/app/controllers/posts_controller.rb]: Delete commented lines

File: blog/test/models/post_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (JesusHerrera) [inline blog/test/models/post_test.rb]: Delete unused file

File: blog/log/development.log [hunk 3]
```diff
@@ -6329,6 +6329,7 @@ Completed 200 OK in 73ms (Views: 71.2ms | ActiveRecord: 0.7ms)
 <<<<<<< HEAD
```
Comment (JesusHerrera) [inline blog/log/development.log]: Delete this file

File: blog/test/controllers/posts_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
```
Comment (JesusHerrera) [inline blog/test/controllers/posts_controller_test.rb]: Aren't you using rspec for tests?

Comment (agusaldasoro) [inline blog/test/controllers/posts_controller_test.rb]: I will, should I delete this line or this file then?

Comment (JesusHerrera) [inline blog/test/controllers/posts_controller_test.rb]: Delete the file

**Comment by JesusHerrera** [comment]
Fix, squash and :+1:

**Pull request merged**